managing_providers/_topics/embedded_workflows.md

```diff
@@ -0,0 +1,339 @@
+      "Error": "FailStateError",
+      "Cause": "No Matches!"
+    },
+
+    "SuccessState": {
+      "Type": "Succeed"
+    }
+  }
+}
+```
+
+### Viewing workflow details
+
+Once a workflow is imported, you can view the details, such as to verify the payload. To view the details for a workflow:
+
+1. Go to the menu: **Automation** > **Embedded Workflows**. Then, click **Workflows** to see the workflows list.
+
+2. Click a workflow. The default *Text* view of the workflow opens where you see the code content.
+
+   ![Text view](../images/embedworkflow_textview.png)
+
+   Alternatively, click the **Graph** tab to see a graph of the same data.
+
+   ![Graph view](../images/embedworkflow_graphView.png)
+
+3. If your workflow has credential fields, create a workflow credential for each of the fields. For more information about creating a credential, see [Adding Credentials](#adding-credentials).
+
+4. Map any workflow credentials to the workflow.
+
+   1. Go to the menu: **Automation** > **Embedded Workflows**. Then, click **Workflows** to see the workflows list.
+
+   2. Select the workflow that includes the credentials that you need to map.
+
+   3. Click **Configuration** > **Map Credentials to this Workflow**.
+
+      The table that displays on the *Map Credentials* page displays all the current mappings for the selected workflow. Verify that the list of credential identifiers includes all of the identifiers that are in the workflow payload.
+
+   4. Use the drop-down fields to map one or more credentials for the workflow. Verify that the table is updated with the correct information.
+
+   5. Click **Save**.
+
+### Creating a Service Dialog using an Embedded Workflow
+
+Fields that are *Dynamic* offer the choice of *Embedded Workflows* in addition to the original *Embedded Automate*.
+
+Dialogs are the part of {{ site.data.product.title_short }} that you interface with when you order services.
+
+1. To open a service dialog, click **Automation** > **Embedded Automate** > **Customization**.
+2. From the *Service Dialog* section, review the list of created dialogs. Click one of the dialogs to view the details.
+3. Click **Edit the dialog**. You can edit the *Name*, *Provider*, and *Template*.
+
+   When you select to edit any of the dropdown settings, you can see a *dynamic* option. Toggle to **On** to make sure it is enabled.
+
+4. Click the **Options** tab. You can now select between *Embedded Automate* and *Embedded Workflows* in the Automation field.
+
+   ![Dialog Dynamic](../images/embedworkflow_servicedialogdynamic.png)
+
+   If you choose **Embedded Workflows**, you can select the workflow to run. Complete the fields, such as *Template* (using drag and drop functionality) and click **Save** when you complete the form. The workflow now runs and refreshes the fields with data from the workflow. In this way fields are chained together. When you *order* a service catalog, the workflow runs in the background to refresh the template.
```

Comment:
> Complete the fields, such as *Template* (using drag and drop functionality) and click **Save** when you complete the form.

Don't think we should call out Template specifically since that was just an example: `Complete the rest of the fields and click **Save** when you complete the form.`

> The workflow now runs and refreshes the fields with data from the workflow.

The workflow won't run when you click save you have to order the service first, you cover this later on so maybe just drop this sentence?

> In this way fields are chained together.

This feels like part of a though, maybe `You can chain dialog fields together by using **Fields to refresh**, this allows one dialog field to kick off the run of another if there are inter-dependencies`

> When you *order* a service catalog, the workflow runs in the background to refresh the template.

Again the template was just an example, `When you *order* a service catalog any dynamic dialogs that are backed by embedded workflows will be run in the background to populate the dialogs.`

Reply:
I have updated this content as suggested.